articles/tech-resources/Changes-to-API-HTTP-Responses-and-Error-Codes

```diff
@@ -0,0 +1,3313 @@
+---
+title: Upcoming Changes to SparkPost API HTTP Responses and Error Codes
+description: Details on upcoming changes that will occur on January 12, 2019
```

Comment:
1) Why was this whole file removed?
2) Need to reword this as possible change as we are "reevaluating" in May, does that mean we should remove the "Post Change" responses and just have this as an error code documentation? If we are re-evaluating in May, but not set in stone, should just not have any actual error code responses in it, but be a simple high level document of proposed changes?

Reply:
1.Was recommended to me to delete the original off of github and to create a new one.
2. okay